daprdocs/content/en/operations/monitoring/tracing/open-telemetry-collector-appinsights.md

```diff
@@ -0,0 +1,88 @@
+---
+type: docs
+title: "Using OpenTelemetry Collector to collect traces to send to AppInsights"
+linkTitle: "Using the OpenTelemetry for Azure AppInsights"
+weight: 1000
+description: "How to use Dapr to push trace events to Azure Application Insights, through the OpenTelemetry Collector."
+---
+
+Dapr can integrate with [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) using the Zipkin API. This guide walks through an example to use Dapr to push trace events to Azure Application Insights, through the OpenTelemetry Collector.
```

Comment:
Suggest 
"Dapr integrates with"
"This guide walks through an example using Dapr to push trace events to Azure Application Insights, using the OpenTelemetry Collector.

Reply:
Done